packages/cli/generation/ir-generator/src/__test__/__snapshots__/openapi/code-samples-open-api.txt

```diff
@@ -144,51 +144,7 @@ exports[`generate IR from OpenAPI 1`] = `
           "errors": [],
           "examples": [
             {
-              "codeSamples": [
-                {
-                  "code": "curl -X GET "http://localhost-custom:8080/telemetry?anonymize=true"
-  -H "accept: application/json"
-  -H "Authorization: Bearer token"",
-                  "docs": null,
-                  "name": null,
-                  "sdk": "curl",
-                  "type": "sdk",
-                },
-                {
-                  "code": "import requests
-url = "http://localhost-custom:8080/telemetry"
-querystring = {"anonymize":"true"}
-headers = {
-    "accept": "application/json",
-    "Authorization": "Bearer token"
-}
-response = requests.request("GET", url, headers=headers, params=querystring)
-print(response.text)",
-                  "docs": null,
-                  "name": null,
-                  "sdk": "python",
-                  "type": "sdk",
-                },
-                {
-                  "code": "import axios from 'axios';
-const options = {
-  method: 'GET',
-  url: 'http://localhost-custom:8080/telemetry',
-  params: {anonymize: 'true'},
-  headers: {
-    accept: 'application/json',
-    Authorization: 'Bearer token'
-  }
-};
-axios.request(options).then(function (response) {
-  console.log(response.data);
-});",
-                  "docs": null,
-                  "name": null,
-                  "sdk": "typescript",
-                  "type": "sdk",
-                },
-              ],
```

Comment:
@armandobelardo this is weird, why is this happening? some customers rely on this functionality

Reply:
Fixed!